yang/frr-bgp-route-map.yang

```diff
@@ -356,6 +356,90 @@ module frr-bgp-route-map {
     }
   }
 
+  typedef route-distinguisher {
+    type string {
+      pattern
+        '(0:(6553[0-5]|655[0-2][0-9]|65[0-4][0-9]{2}|'
```

Comment:
The question is: is this maintainable overall? :smile:

Reply:
I am not yang expert enough to know if I can make inheritance between typedefs.
What I just do is to enlarge the regexp, hoping that one day, the ietf will follow. :)